src/resources/postcss/tickets-v1.pcss

```diff
@@ -166,10 +166,13 @@
 			}
 		}
 
+		/* V1 views had a typo in "suffix" so we cover both. */
 		&.tribe-tickets__item--price-sufix,
-		&.tribe-tickets__item--price-sufix.show-optout {
-			@media(--viewport-medium) {
-				-ms-grid-columns:  5.5fr 3fr;
+		&.tribe-tickets__item--price-sufix.show-optout,
+		&.tribe-tickets__item--price-suffix,
+		&.tribe-tickets__item--price-suffix.show-optout {
```

Comment:
If we're updating the V1 templates for this fix, do we want to keep `sufix`? Where's it used?